kevm-pyk/src/kevm_pyk/cli.py

```diff
@@ -850,6 +852,13 @@ def kprove_args(self) -> ArgumentParser:
             action='store_true',
             help='Use the implication check of the Booster (experimental).',
         )
+        args.add_argument(
+            '--optimize-kcfg',
+            dest='optimize_kcfg',
+            default=None,
+            action='store_true',
+            help='Optimize the constructed KCFG on-the-fly.',
```

Comment:
Can we make this a bit more detailed?
```suggestion
            help='Optimize the constructed KCFG on-the-fly by minimizing consecutive edges.',
```

Reply:
That's what it is now, but we don't know what else it will be in the future.